independent-publisher-connectors/Proposal - RegEx Matching (Independent Publisher)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,466 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "RegEx Matching",
+    "description": "This connector checks whether a text string matches a specified pattern.",
+    "version": "1.0",
+    "contact":{
+      "name": "Mitanshu Garg",
+      "url": "https://twitter.com/mitanshu",
+      "email": "garg.mitanshu@gmail.com"
+    }
+  },
+  "host": "sa018.azurewebsites.net",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "x-ms-connector-metadata":[
+    {
+      "propertyName":"Website",
+      "propertyValue":"https://sa018.azurewebsites.net"
+    },
+    {
+      "propertyName":"Privacy Policy",
+      "propertyValue":"https://sa018.azurewebsites.net"
+    },
+    {
+      "propertyName":"Categories",
+      "propertyValue":" Content and Files"
+    }
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/validEmail/": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",

```

Comment:
@happyume please add a more detailed description for your 2xx code for all operations, `default` description doesn't fit here. You can refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses) for an example.

Reply:
This is now updated